packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -67,13 +69,14 @@ abstract class Intent with Diagnosticable {
 /// To register an action listener, call [Action.addActionListener].
 typedef ActionListenerCallback = void Function(Action<Intent> action);
 
-/// Base class for actions.
+/// Base class for an action or command to be performed.
+///
+/// {@youtube 560 315 https://www.youtube.com/watch?v=XawP1i314WM}
```

Comment:
What does `560 315` mean? Is that the width and height of the video or something?

Should we add some guidance on the wiki? https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#documentation-dartdocs-javadocs-etc

Reply:
Yes, it's the width and height of the video window. And yes, it would be good to add that to the wiki.

Reply:
Added: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#link-to-widget-of-the-week-videos